Projects/Claims/Sources/Models/FlowClaimSingleItemCheckoutStepModel.swift

```diff
@@ -50,4 +50,20 @@ struct ClaimFlowMoneyModel: Codable, Equatable {
         formatter.currencySymbol = self.currencyCode
         return formatter.string(for: amount) ?? ""
     }
+
+    func formatAndPrintDouble(value: Double) -> String {
+        let formatter = NumberFormatter()
+        formatter.groupingSeparator = " "
+        formatter.numberStyle = .decimal
```

Comment:
Try switching it to .currency and set currencySymbol for formatter.
You don't need to send value since we can do any operation on amount property inside structure.
Then, this function will return something like 1.200 SEK